pkg/webui/console/store/actions/application.js

```diff
@@ -46,68 +30,66 @@ import {
 } from './api-key'
 
+import { createRequestActions } from './lib'
+
 export const SHARED_NAME = 'APPLICATION'
 
 export const GET_APP_BASE = 'GET_APPLICATION'
-export const GET_APP = `${GET_APP_BASE}_REQUEST`
-export const GET_APP_SUCCESS = `${GET_APP_BASE}_SUCCESS`
-export const GET_APP_FAILURE = `${GET_APP_BASE}_FAILURE`
-export const GET_APP_API_KEYS_LIST = createGetApiKeysListActionType(SHARED_NAME)
-export const GET_APP_API_KEYS_LIST_SUCCESS = createGetApiKeysListSuccessActionType(SHARED_NAME)
-export const GET_APP_API_KEYS_LIST_FAILURE = createGetApiKeysListFailureActionType(SHARED_NAME)
-export const GET_APP_API_KEY = createGetApiKeyActionType(SHARED_NAME)
-export const GET_APP_API_KEY_SUCCESS = createGetApiKeySuccessActionType(SHARED_NAME)
-export const GET_APP_API_KEY_FAILURE = createGetApiKeyFailureActionType(SHARED_NAME)
-export const GET_APP_COLLABORATOR_PAGE_DATA = createGetCollaboratorActionType(SHARED_NAME)
-export const GET_APP_COLLABORATORS_LIST = createGetCollaboratorsListActionType(SHARED_NAME)
-export const GET_APP_COLLABORATORS_LIST_SUCCESS = createGetCollaboratorsListSuccessActionType(SHARED_NAME)
-export const GET_APP_COLLABORATORS_LIST_FAILURE = createGetCollaboratorsListFailureActionType(SHARED_NAME)
+export const [{
+  request: GET_APP,
+  success: GET_APP_SUCCESS,
+  failure: GET_APP_FAILURE,
+}, {
+  request: getApplication,
+  success: getApplicationSuccess,
+  failure: getApplicationFailure,
+}] = createRequestActions(GET_APP_BASE,
+  id => ({ id }),
+  (id, selector) => ({ selector })
+)
+
+export const GET_APP_API_KEYS_LIST_BASE = createGetApiKeysListActionType(SHARED_NAME)
+export const [{
+  request: GET_APP_API_KEYS_LIST,
+  success: GET_APP_API_KEYS_LIST_SUCCESS,
+  failure: GET_APP_API_KEYS_LIST_FAILURE,
+}, {
+  request: getApplicationApiKeysList,
+  success: getApplicationApiKeysListSuccess,
+  failure: getApplicationApiKeysListFailure,
+}] = createRequestActions(GET_APP_API_KEYS_LIST_BASE, (appId, { page, limit } = {}) => ({ appId, params: { page, limit }}))
+
+export const GET_APP_API_KEY_BASE = createGetApiKeyActionType(SHARED_NAME)
+export const [{
+  request: GET_APP_API_KEY,
+  success: GET_APP_API_KEY_SUCCESS,
+  failure: GET_APP_API_KEY_FAILURE,
+}, {
+  request: getApplicationApiKey,
+  success: getApplicationApiKeySuccess,
+  failure: getApplicationApiKeyFailure,
+}] = createRequestActions(GET_APP_API_KEY_BASE, (appId, keyId) => ({ entityId: appId, keyId }))
+
+export const GET_APP_COLLABORATORS_LIST_BASE = createGetCollaboratorsListActionType(SHARED_NAME)
+export const [{
+  request: GET_APP_COLLABORATORS_LIST,
+  success: GET_APP_COLLABORATORS_LIST_SUCCESS,
+  failure: GET_APP_COLLABORATORS_LIST_FAILURE,
+}, {
+  request: getApplicationCollaboratorsList,
+  success: getApplicationCollaboratorsListSuccess,
+  failure: getApplicationCollaboratorsListFailure,
+}] = createRequestActions(GET_APP_COLLABORATORS_LIST_BASE, (appId, { page, limit } = {}) => ({ appId, params: { page, limit }}))
```

Comment:
same as for api-keys/api-key. either remove unused exports from `action/collaborators` or check how I proposed to abstract this.